src/views/Peers.tsx

```diff
@@ -99,11 +111,28 @@ export const Peers = () => {
             message.loading({ content: 'Deleting...', key: deleteKey, style });
         } else if (deletedPeer.success) {
             message.success({ content: 'Peer has been successfully removed.', key: deleteKey, duration: 2, style });
+            dispatch(peerActions.resetDeletedPeer(null))
         } else if (deletedPeer.error) {
             message.error({ content: 'Failed to delete peer. You might not have enough permissions.', key: deleteKey, duration: 2, style  });
+            dispatch(peerActions.resetDeletedPeer(null))
         }
     }, [deletedPeer])
 
+    const saveGroupsKey = 'saving_groups';
+    useEffect(() => {
+        const style = { marginTop: 85 }
+        if (savedGroups.loading) {
+            message.loading({ content: 'Updating peer groups...', key: saveGroupsKey, style });
+        } else if (savedGroups.success) {
+            message.success({ content: 'Peer groups have been successfully updated.', key: saveGroupsKey, duration: 2, style });
+            setUpdateGroupsVisible({} as Peer, false)
+            dispatch(peerActions.resetSavedGroups(null))
+        } else if (savedGroups.error) {
+            message.error({ content: 'Failed to update peer groups. You might not have enough permissions.', key: saveGroupsKey, duration: 2, style  });
```

Comment:
Codacy has a fix for the issue: [Replace `······message.error({·content:·'Failed·to·update·peer·groups.·You·might·not·have·enough·permissions.',·key:·saveGroupsKey,·duration:·2,·style··});` with `message.error({⏎········content:⏎··········"Failed·to·update·peer·groups.·You·might·not·have·enough·permissions.",⏎········key:·saveGroupsKey,⏎········duration:·2,⏎········style,`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      message.error({
```